scripts/jenkins-build.bat

```diff
@@ -68,11 +68,32 @@ cmake -G "Ninja" ^
 cmake --build . || exit /b
 cmake --build . --target install || exit /b
 
+REM Build Java
 cd "%WORKSPACE%"
+cd source
+call mvn clean install
```

Comment:
Unsure `call` is appropriate here; it's for invoking another batch file, not a command.  Why not using `mvn clean install || exit /b` to get the appropriate error handling behaviour?  This is equivalent to `set -e` in a shell script.